### DISCUSSION TITLE
Switch Default GPU Render | Hybrid Mode | AMD only

### DISCUSSION THREAD

**Discussion by b1kr3m** [prasanthrangan/hyprdots#1563] [Q&A]
Hey, actually i am using a Gaming Laptop. So the thing is that  @prasanthrangan  Sir dots  automatic set Nvidia Gpu as a Prime renderer, I am not a desktop user so whenever the laptop unplug for any reason it automatic shutdown because of  power switching. I thing iGPU is sufficient for Hyprland, So I just wanna change the render option like Nvidia prime (Hybrid Mode ) or only AMD Gpu only.....

Last time i tried to change it but lack of experience and knowledge i just screw up my system, So Guys I want to your help to do the thing in a secure way. after 3 time breaks my system i have just no more energy left😅, so I need your guidance... 

![image](https://github.com/prasanthrangan/hyprdots/assets/97847882/9da02e19-9d5f-4aa0-9528-2e78c319adf4)

I am attach some information so it will help you guys to understand more accurately ...
![image](https://github.com/prasanthrangan/hyprdots/assets/97847882/724df207-50fa-4386-92d6-bccb3b6da689)

![240604_10h42m20s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/97847882/7e6406f7-7c88-4a7c-970b-5869ec1e42bf)

**Comment by its-19818942118** [comment]
hmm
k. so basically you want hyprland to use amd instead of nvidia is that it?

**Reply by b1kr3m** [reply]
Not actually, I just want to enable Nvidia-prime so it takes dGPU whenever heavy task knock.

**Comment by its-19818942118** [comment]
@Lalit-710Cx

**Comment by its-19818942118** [comment]
so what you need to do is basically define the card yourself on .config//hypr/card. but for that we need some confirmation. do ```lspci | grep -E 'VGA|3D'```paste that output. ```ls -l /dev/dri/by-path``` paste that output too then we can go to the next steps as wrong symlink without being sure can cause hazard. well we dont want that so this just a confirmation. so waht is basically does is that the first command lists all pci objects that renders graphics etc. then the second command just list the paths of those stuff by its identifier aka the numbers on the left side. anyways i am bad at explainging. so paste the output of the above commands

**Reply by b1kr3m** [reply]
![image](https://github.com/prasanthrangan/hyprdots/assets/97847882/f67fd9de-f38b-4543-b0aa-cd0925965c8a)
![image](https://github.com/prasanthrangan/hyprdots/assets/97847882/216fb00e-41c1-43bc-b86c-c9957f020b56)

**Reply by b1kr3m** [reply]
https://wiki.hyprland.org/Configuring/Multi-GPU
I follow this Guidance also but nothing change @its-19818942118 brother.

**Reply by its-19818942118** [reply]
> ![image](https://private-user-images.githubusercontent.com/97847882/337212302-f67fd9de-f38b-4543-b0aa-cd0925965c8a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc3Njc3MTAsIm5iZiI6MTcxNzc2NzQxMCwicGF0aCI6Ii85Nzg0Nzg4Mi8zMzcyMTIzMDItZjY3ZmQ5ZGUtZjM4Yi00NTQzLWIwYWEtY2QwOTI1OTY1YzhhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA3VDEzMzY1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY1ZmZiYTJjNGYyZTljYjk0MDYwYWRjYTU5OWZiYjhlNDQyMzBmMGRhODgyODJhYjMxM2JlMjVhZTNhYjFiNjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5PmIMBYt-U3WWhbE5N69Y3p9R5JcW9Q23bw19GPDfxo) ![image](https://private-user-images.githubusercontent.com/97847882/337212466-216fb00e-41c1-43bc-b86c-c9957f020b56.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc3Njc3MTAsIm5iZiI6MTcxNzc2NzQxMCwicGF0aCI6Ii85Nzg0Nzg4Mi8zMzcyMTI0NjYtMjE2ZmIwMGUtNDFjMS00M2JjLWI4NmMtYzk5NTdmMDIwYjU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA3VDEzMzY1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc4NWQ5YzkzZWI3Mzc3M2YyMzljNjM1MzhmMjIwMTI3YjllNTFjM2QwODc2ZDNmNmNiNjkyZjIyZmE4MzEzM2ImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.JKMUUssGMDAbLtcUOh6l4GcC2Q7GB7sA9HWpGKO7oWU)

damn welp not my area of expertise. well i am a noob in a sense sooo.

**Comment by kRHYME7** [comment]
https://wiki.archlinux.org/title/NVIDIA
https://wiki.archlinux.org/title/NVIDIA_Optimus

Check the Arch first wiki  and see what works best for you

**Reply by b1kr3m** [reply]
Ok....got it.
Nice to see you again @kRHYME7 
You are inactive in the discord group for more than a week....
Now u are back again🤙

**Reply by kRHYME7** [reply]
I worked on some rushed project. 😷 It's officially over today. 😅 Need to regen. 😴💤
